components/tests/ui/testcases/web/search.txt

```diff
@@ -0,0 +1,378 @@
+*** Settings ***
+Documentation     Tests Search on Web
+...               https://www.openmicroscopy.org/private/ome-internal/testing_scenarios/Search.html
+
+# Resource          ../config.txt
+Resource          ../../resources/config.txt
+Library           DateTime
+Library           Selenium2Library      run_on_failure=Nothing
+
+*** Variables ***
+
+${XpathImage}           xpath=//table[@id='dataTable']//img[@alt='image']
+${XpathDataset}         xpath=//table[@id='dataTable']//img[@alt='dataset']
+${XpathProject}         xpath=//table[@id='dataTable']//img[@alt='project'] 
+${XpathPlate}           xpath=//table[@id='dataTable']//img[@alt='plate']
+${XpathTable}           xpath=//table[@id='dataTable']
+${XpathTableColumn}     xpath=//table[@id='dataTable']//tr/td[position()=3]
+${XPathAlertText}       xpath=//div[contains(@class,'ui-dialog')][contains(@style,'display: block')]//p
+${RandomClick}          xpath=//form[@id="searching_form"]/h2
+${AlertWindow}          xpath=//div[contains(@class,'ui-dialog')][contains(@style,'display: block')]//button/span[contains(text(),"OK")]
+
+${Search_Query}         1
+${Search_Query1}        test
+${Default_Srch_Group}   All Groups
+${Default_Srch_Date}    Import date
+${Table_Header}         Name
+${ImplicitDELAY}        0.5
+${DayStartTime}         00:00:00
+${DayEndTime}           2014-07-18 00:00:00
+${FromDate}             2014-01-15
+${FromDateWithTime}     2014-01-15 00:00:00
+${EndDate}              2014-07-17
+${WrongStartDate}       2014-07-20
+${AcquiredDateCol}      4
+${NotAnImage}           3
+${alertText}            Wildcard searches (*) must contain more than a single wildcard.
+
+${SEARCH URL}           http://${WEB HOST}${WEB PREFIX}/webclient/search/
```

Comment:
This is failing on a non HTTP web with https://ci.openmicroscopy.org/view/Failing/job/OMERO-5.1-merge-robotframework/183/console
```
05:30:57 Test Default Search Check :: login,search and Check that you are t... | FAIL |
05:30:57 Location should have been 'http://trout.openmicroscopy.org/integration/webclient/search/' but was 'https://trout.openmicroscopy.org/integration/webclient/search/'
```

`${WELCOME URL}` should be used instead of `http://${WEB HOST}${WEB PREFIX}/webclient`